index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+    <title>mobile-view</title>
+    
+    <link rel="stylesheet" href="style.css">
+    
+</head>
+<body>    
+    <main>
+        <header>
+        <div class="wrapper1">
+          <div class="header-wrapper">
+            <div class="logo"><a href="index.html">My Logo</a></div>
+            <nav>
+              <a href="">
+                <img id="menu1" src="images/menu1.png" alt="menu" />
+              </a>
+            </nav>
+          </div>
+        
+          <section class="details">
+            <h1>I'm Yusuf Glad to see you!</h1>
+            <p>
+              I’m a software developer! I can help you build a product , feature
+              or website Look through some of my work and experience! If you like
+              what you see and have a project you need coded, don’t hestiate to
+              contact me.
+            </p>
+            <a id="connect" href="index.html">LET'S CONNECT</a>
+            <ul class="details-list1">
+              <li>
+                <a href=""> <img src="images/twitter.png" alt="Twitter" /></a>
+              </li>
+              <li>
+                <a href=""> <img src="images/Linkedin.png" alt="LinkedIn" /></a>
+              </li>
+              <li>
+                <a href=""> <img src="images/medium.png" alt="Media" /></a>
+              </li>
+              <li>
+                <a href=""> <img src="images/Vector-1.png" alt="image" /></a>
+              </li>
+              <li>
+                <a href=""> <img src="images/Vector-3.png" alt="image" /></a>
+              </li>
+            </ul>
+          </section>
```

Comment:
- [ ] Please move this whole section outside the `<header>` tag because the `<header>` tag should include the logo and the hamburger button only.